www/content/docs.md

```diff
@@ -152,36 +152,79 @@ For npm-style build systems, you can install htmx via [npm](https://www.npmjs.co
 npm install htmx.org@2.0.1
 ```
 
-After installing, you’ll need to use appropriate tooling to use `node_modules/htmx.org/dist/htmx.js` (or `.min.js`).
-For example, you might bundle htmx with some extensions and project-specific code.
+After installing, you’ll need to use appropriate tooling to use `node_modules/htmx.org/dist/htmx[.esm, .cjs, .amd].js` (or `.min.js`).
 
-### Webpack
+In the simplest case all you need to do is to add this import to your entry point (like `index.js`)
+```js
+import 'htmx.org';
+```
+
+However, you will probably need the global `htmx` variable, which gives your code access to methods like
```

Comment:
If I may offer a suggestion:

The use of "probably" is unclear to beginners; You give a very concrete example right after (the use of htmx plugins), but is this the only scenario?